firefox-ios/firefox-ios-tests/Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -246,8 +244,7 @@ class ActivityStreamTest: BaseTestCase {
     }
 
     private func checkNumberOfExpectedTopSites(numberOfExpectedTopSites: Int) {
-        mozWaitForElementToExist(app.cells[TopSites.itemCell])
-        mozWaitForElementToExist(app.cells[TopSites.itemCell])
+        mozWaitForElementToExist(app.cells[TopSites.itemCell], timeout: 20)
```

Comment:
The default timeout is 20 so the `timeout:` argument isn't needed.

Reply:
Fixed in faf7ad1bbd4eddfce845f870d8250af73ee0fd39